models/acronym.js

```diff
@@ -0,0 +1,16 @@
+const acronym = (sequelize, DataTypes) => {
+    constAcronym = sequelize.define('acronym', {
+        acronym: {
+            type: DataTypes.STRING
+        },
+        name: {
+            type: DataTypes.STRING
+        },
+        notes: {
+            type: DataTypes.STRING
```

Comment:
I think this should be `DataTypes.TEXT`. Strings are limited to 255 characters, which might not be enough for lengthy notes. See more here - http://docs.sequelizejs.com/manual/tutorial/models-definition.html

Reply:
OK great, I'll read up.  Thanks!